README.md

```diff
@@ -1,97 +1,142 @@
+  ```bash
+  */20 * * * * cd /path/to/journalduhacker && env RAILS_ENV=production bundle exec rake ts:index > /dev/null
+  ```
+## Moderation
 
-* (Optional, only needed for the search engine) Install Sphinx.  Build Sphinx
-config and start server:
+On-site tasks are carried out directly on the website. Console tasks are carried out through the Ruby on Rails console in production.
+To start the Rails console: `rails c`. To start the Rails console and rollback on exit: `rails c --sandbox`.
 
-          journalduhacker$ rake ts:rebuild
+Note: when moderating you should provide a moderation reason.
 
-* Define your site's name and default domain, which are used in various places,
-in a `config/initializers/production.rb` or similar file:
+### On-site tasks
 
-          class << Rails.application
-            def domain
-              "example.com"
-            end
-          
-            def name
-              "Example News"
-            end
-          end
-          
-          Rails.application.routes.default_url_options[:host] = Rails.application.domain
+#### Edit/Delete a story
 
-* Put your site's custom CSS in `app/assets/stylesheets/local`.
+Click the 'edit' button under the story title. You can:
+* Delete the story
+* Edit the URL
+* Edit the title
+* Add or remove tags
+* Edit the body text
+* Merge the story into another
 
-* Seed the database to create an initial administrator user and at least one tag:
+#### Delete a comment
 
-          journalduhacker$ rake db:seed
-          created user: test, password: test
-          created tag: test
+Click the 'delete' button next to it.
 
-* Run the Rails server in development mode.  You should be able to login to
-`http://localhost:3000` with your new `test` user:
+#### Attach a moderation note to a user
 
-          journalduhacker$ rails server
+Inside a user profile page, you can write moderation notes for that particular user. Only moderators can see them.
 
-* In production, set up crontab or another scheduler to run regular jobs:
+#### Disable invites for a user
+
+Inside a user profile page, click the 'Disable Invites' button at the bottom.
+You will be able to re-enable invites by clicking on 'Enable Invites'.
+
+#### Ban a user
+
+Inside a user profile page, click the 'Ban' button at the bottom. You must provide a reason.
+You will be able to unban the user by clicking on 'Unban'.
+
+#### Read the latest 10 moderation notes and moderations applied to a user
+
+You can see them inside a user profile page. To see more than 10, you need to use the console.
+
+#### Manage hats requests
+
+Go to /hats/requests.
+
+### Console tasks
+
+A typical command to add a resource is:
+```ruby
+name_of_resource.create(attribute1: value1, attribute2: value2, ...)
+```
+
+To edit a resource first assign it to a variable, edit it and then save:
+```ruby
+story = Story.find_by(short_id: story_short_id)
+story.attribute = new_attribute_value
+story.save
```

Comment:
```suggestion
story.save()
```
In ruby sintatticamente è corretto ma per coerenza con story.delete() scritto sotto e per leggibilità metterei le parentesi tonde.